composer.json

```diff
@@ -29,14 +29,15 @@
     "symfony/process": "~2.1||~4.1.0",
     "symfony/yaml": "^3.3||^4.0",
     "twig/twig": "^1.0||^2.0",
-    "magento/magento-cloud-components": "^1.0.1"
+    "magento/magento-cloud-components": "^1.0.1",
+    "magento/magento-cloud-patches": "^1.0"
```

Comment:
why for `cloud-components` we allow minor releases and for `patches` only patch releases? 
I am not against this, I just want to know the reason of this decision.

Reply:
This is the same as `^1.0.0`, so until `1.0.1` is released it does not make sense to add patch constraint part

Reply:
https://semver.mwl.be/#!?package=madewithlove%2Felasticsearcher&version=%5E0.5&minimum-stability=stable vs https://semver.mwl.be/#!?package=madewithlove%2Felasticsearcher&version=%5E0.5.0&minimum-stability=stable

Reply:
I mixed up with `^0.1` which is equal to `<=0.1.0 <0.2.0`

Reply:
Btw, I did not got what you wanted to show with those links. They do not show the real rule from this constraint

Reply:
I just showed there is no difference betweeen `^1.0` and `^1.0.0` at this point